packages/about-you/api-client/package.json

```diff
@@ -1,19 +1,19 @@
 {
   "name": "@vue-storefront/about-you-api",
-  "version": "0.0.1",
+  "version": "0.1.0",
   "private": true,
   "sideEffects": false,
   "main": "lib/index.cjs.js",
   "module": "lib/index.es.js",
-  "types": "lib/index.d.ts",
+  "types": "lib/about-you/api-client/src/index.d.ts",
```

Comment:
Is this path correct? Rest of the index files are in lib/index*.

Reply:
Yes, for some reason it is. I double-checked it with @filrak

Reply:
Well. Ok, then